textures/src/main/java/com/dumbdogdiner/stickyapi/util/textures/headdatabase/MinecraftHeadsAPI.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2020-2021 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.util.textures.headdatabase;
+
+import com.dumbdogdiner.stickyapi.common.ServerVersion;
+import com.dumbdogdiner.stickyapi.util.http.HttpException;
+import com.dumbdogdiner.stickyapi.util.textures.InvalidTextureException;
+import com.dumbdogdiner.stickyapi.util.textures.TextureHelper;
+import com.dumbdogdiner.stickyapi.webapi.minecraftheads.MinecraftHeadsWebAPI;
+import lombok.experimental.UtilityClass;
+import me.arcaniax.hdb.api.HeadDatabaseAPI;
+
+import org.bukkit.Bukkit;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+/**
+ * A simple basic interface to the minecraft-heads API plugin
+ */
+@UtilityClass
+public class MinecraftHeadsAPI {
+    private static final boolean usePlugin;
+    private static final HeadDatabaseAPI mcheads;
+
+    static {
+        //noinspection ConstantConditions
+        if(ServerVersion.isBukkit() && Bukkit.getServer() != null) {
+            if (Bukkit.getPluginManager().getPlugin("HeadDatabase") == null) {
+                Bukkit.getLogger().severe("HeadDatabase Plugin was not found!");
```

Comment:
Definitely some logic simplifying possible here - you set them null if none of these conditions match?

Reply:
If we don't have the heads database plugin availible (either because its not installed or because we arent on bukkit) then it goes to fallback